tests/SBRPTests/PackageData.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace SBRPTests;
+
+public class PackageData
```

Comment:
Do you see this being used across multiple tests?  If not consider utilizing MemberData via a class instance within the consuming test case.  Getting the test data closer to the place where it is used helps improve the UX/maintainability.